modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -30,13 +30,20 @@ image::op-pipeline-yaml.png[]
 Configure your pipeline using the *Pipeline Builder*:
 
 .. In the *Name* field, enter a unique name for the pipeline.
-.. In the *Tasks* section, select a task from the *Select Task* list to add a task to the pipeline. In this example, use the *s2i-nodejs* task.
-*** To add sequential tasks to the pipeline, click the plus icon to the right or left of the task, and from the *Select Task* list, select the task you want to add to the pipeline. For this example, use the plus icon to the right of the *s2i-nodejs* task to add an *openshift-client* task.
-*** To add a parallel task to the existing task, click the plus icon displayed next to the task, and from the *Select Task* list, select the parallel task you want to add to the pipeline.
+.. In the *Tasks* section, click *Add task*, search for a task in the quick search, select the required task from the list and click *Add* or *Install and add*. In this example, use the *s2i-nodejs* task.
++
+[NOTE]
+====
+The search list contains all the Tekton Hub tasks and tasks available on the cluster. Also, if a task is already installed it will show *Add* to add the task whereas it will show *Install and add* to install and add the task.
```

Comment:
Should we also mention the upgrade path as well in the note ?

Update Path:
If a specific version of a task (eg: 0.1) is already installed on the cluster and when the user chooses to update the task version to 0.2, then "Update and add" option will be available in the UI with a warning note as below.

<img width="845" alt="Screenshot 2021-09-22 at 9 44 15 AM" src="https://user-images.githubusercontent.com/9964343/134282507-f5cad7e6-e9b6-4b31-a8c0-1d9a63393862.png">

Reply:
Thanks for your input @karthikjeeyar I'll try to repurpose the content with the suggestion.